Grammar/DefuseGrammar.cs

```diff
@@ -216,4 +216,4 @@ private static Grammar _WhoIsOnFirstGrammar()
             return new Grammar(new GrammarBuilder(new Choices(File.ReadAllLines(@"WhoIsOnFirst.txt")))) {Name = "Who's On First Grammar"};
         }
     }
-}
```

Comment:
Any reason in particular for this whitespace? Or did you just forget it?

Reply:
No particular reason. My bad.